README.md

```diff
@@ -67,6 +67,7 @@ nano.socket('bus', { raw: true } );
 * `'rcvmaxsize'` *(Number, default: `false`)*: see [`socket.rcvmaxsize(size)`](https://github.com/nickdesaulniers/node-nanomsg#socketrcvmaxsizesize)
 * `'chan'` *(Array, default: `['']`)*: see [`socket.chan(Array)`](https://github.com/nickdesaulniers/node-nanomsg#socketchanarray)
 * `'wsopt'` *(String, default: `'binary'`)*: see [`socket.wsopt(str)`](https://github.com/nickdesaulniers/node-nanomsg#socketwsoptstr)
+* `'dontwait'` *(Number, default: `'1'`)*: see [`socket.dontwait(n)`](https://github.com/nickdesaulniers/node-nanomsg#socketdontwaitn)
```

Comment:
yea i thought about making it a boolean and wasn't sure which to go with, so if you have an opinion, i'll definitely make this bool and update